index.bs

```diff
@@ -12031,7 +12031,22 @@ class Vec3 {
 
 <h2 id="changelog">
 Change Log</h2>
-<h3 id="changestart">
+<h3 id="changestart1">
+Since Working Draft of 19 June 2018</h3>
+* Update implementation-report.html
+* Widen range for detune values
+* PannerNode constructor throws errors
+* Rephrase algorithm for setting buffer and curve
+* Refine startRendering algorithm.
+* Spec more precisely events overlapping with SetValueCurveAtTime
```

Comment:
"Spec more precisely events" reads better as
"Spec, more precisely, events" or "Spec events ... SetValueCurveAtTime more precisely".

Maybe we shouldn't say "Spec" but "Specify"

Reply:
agreed, just copying from the git logs, guess I should have read some of them a bit more closely